c7n/offhours.py

```diff
@@ -226,15 +224,15 @@ def process_current_time(self, i, parts):
         now = datetime.datetime.now(tz).replace(
             minute=0, second=0, microsecond=0)
 
-        if not self.weekends and now.weekday() in (5, 6):
-            log.debug("skipping weekends")
-            return False
-
         #override get_sentinel_time with get_custom_time if custom downtime
         #detected
         if self.is_custom(parsed):
             return self.get_custom_time(i, now, parsed)
 
+        if not self.weekends and now.weekday() in (5, 6):
```

Comment:
Moved the weekend check to be after the `is_custom` check. If the value of the tag isn't a custom schedule we will want to default to existing logic.